package.json

```diff
@@ -43,6 +43,7 @@
         "onCommand:language-julia.debug.getActiveJuliaEnvironment",
         "onLanguage:julia",
         "onLanguage:juliamarkdown",
+        "onWebviewPanel:JuliaDocumentationBrowser",
```

Comment:
I think I would skip this whole part, it seems reasonable to me that the doc pane doesn't stay open across VS Code shutdowns.

Reply:
hm, I'm honestly against this, since some users would like to see a doc pane in a specific location, which might be different from the default location, and then she wants to keep it in a same location across sessions. I think many Juno users appreciate how Juno preserve the layout in the last session.
Why do you want to skip this ?